.github/workflows/build.yaml

```diff
@@ -266,34 +280,38 @@ jobs:
       if: ${{ env.SSH_PRIVATE_KEY == '' }}
       run: git clone https://gitlab.com/OpenMW/openmw-deps.git
 
-    - name: Move exported vcpkg packages and pdb files archive to openmw-deps repository
-      run: mv vcpkg-x64-*-${{ github.sha }}/vcpkg-x64-*-${{ github.sha }}.7z openmw-deps/windows/
+    - name: Generate files containing a download url and hash
+      run: |
+        for file in windows-*-${{ github.ref_name }}.7z; do
+          echo "${{ github.server_url }}/${{ github.repository }}/releases/download/${{ github.ref_name }}/$file" > "openmw-deps/windows/$file.txt"
+          shasum -a 512 "$file" >> "openmw-deps/windows/$file.txt"
+        done
 
     - name: Generate commit message
       env:
         GH_TOKEN: ${{ github.token }}
       working-directory: ${{ github.workspace }}/openmw-deps
       run: |
-        echo Add vcpkg packages built at ${{ github.sha }} > commit_message.txt
+        echo Add vcpkg package urls built at ${{ github.sha }} > commit_message.txt
         echo >> commit_message.txt
         printf 'Pushed by ' >> commit_message.txt
         url=$(gh run --repo ${{ github.repository }} view ${{ github.run_id }} --json jobs --jq '.jobs[] | select(.name == "push-dynamic") | .url')
         echo "${url:?}" >> commit_message.txt
         echo >> commit_message.txt
 
-    - name: Commit exported vcpkg packages
+    - name: Commit generated metadata
       working-directory: ${{ github.workspace }}/openmw-deps
       run: |
         git checkout -b vcpkg-x64-${{ github.sha }}
-        git add windows/vcpkg-x64-*-${{ github.sha }}.7z
+        git add windows/windows-*-${{ github.ref_name }}.7z.txt
```

Comment:
Having windows in the directory and file name is repetitive. Why do you change the naming schema? What's wrong with `vcpkg-x64-*-${{ github.ref_name }}.7z.txt`?

Reply:
It changed because I cut all the gitlab stuff and then didn't change it back after. Should be back to normal now.